netsuite/restlet.py

```diff
@@ -30,7 +30,7 @@ def hostname(self):
     def request(
         self,
         script_id: int,
-        payload: Any = None,
+        payload: Any = {},
```

Comment:
You don't want to use an empty dict as a default value; you can actually cause defaults to be mutated in the course of function calls that way! See 

https://repl.it/repls/SpiffyFullLicense

```
class A:
  def some_function(self, *add_stuff, default_thing={}):
    default_thing.update({'blarg': 'arggg'})
    if [*add_stuff]:
      default_thing.update({'added_stuff':[*add_stuff]})
    print(default_thing)

a = A()
a.some_function()
a.some_function(1,2,3,4,5)
a.some_function()

# outputs 
#{'blarg': 'arggg'}
#{'blarg': 'arggg', 'added_stuff': [1, 2, 3, 4, 5]}
#{'blarg': 'arggg', 'added_stuff': [1, 2, 3, 4, 5]}
# because we actually mutated the default object!

class B:
  def some_function(self, *add_stuff, default_thing=None):
    if default_thing is None:
      default_thing = {}
    default_thing.update({'blarg': 'arggg'})
    if [*add_stuff]:
      default_thing.update({'added_stuff':[*add_stuff]})
    print(default_thing)

b = B()
b.some_function()
b.some_function(1,2,3,4,5)
b.some_function()

# outputs
#{'blarg': 'arggg'}
#{'blarg': 'arggg', 'added_stuff': [1, 2, 3, 4, 5]}
#{'blarg': 'arggg'}
# all good
```